packages/vuetify/src/composables/defaults.ts

```diff
@@ -39,6 +40,10 @@ export function provideDefaults (
   const providedDefaults = ref(defaults)
 
   const newDefaults = computed(() => {
+    const disabled = unref(options?.disabled)
+
+    if (disabled) return {}
```

Comment:
Should this be `return injectedDefaults.value`? Currently `disabled` blocks defaults propagation entirely

```html
<v-defaults-provider :defaults="{ VIcon: { color: 'red' }}">
  <v-icon icon="mdi-vuetify" />
  <v-chip prepend-icon="mdi-vuetify" />
  <v-chip>
    <template #prepend>
      <v-icon icon="mdi-vuetify" />
    </template>
  </v-chip>
</v-defaults-provider>
```